boards/airmind/mindpx-v2/init/rc.board_sensors

```diff
@@ -14,3 +14,6 @@ mpu6000 -s -R 8 start
 mpu9250 -s -R 8 start
 lsm303d -R 10 start
 l3gd20 -R 14 start
+
+# Internal SPI (auto detect ms5611 or ms5607)
+ms5611 -T 0 -s start
```

Comment:
Let's skip the autodetect except on fmu-v2 and fmu-v3 (the only place I've seen an ms5607).

Reply:
Done